scalars/contributed/jakobmerrild/long.md

```diff
@@ -0,0 +1,79 @@
+
+| Output                  | Explanation                                                              |
+| ----------------------- | ------------------------------------------------------------------------ |
+| `+1234`                 | Leading `+` is not allowed                                               |
+| `-10223372036854775808` | Value is lower than `-2^63`                                              |
+| `12223372036854775807`  | Value is greater than `2^63-1`                                           |
+| `123.0`                 | Fractional part is not allowed even if it is zero                        |
+| `1e6`                   | Exponential notation is not allowed, even if it represents a valid value |
+| `"12345"`               | String representations of a valid value are not allowed                  |
+
+# Literal Input spec
+
+For input both IntValue and StringValue shall be accepted so long as the StringValue is a base-10 representation of a
+valid value within the range.
+
+These are valid examples:
+
+| Input                  | Explanation                                                               |
+| ---------------------- | ------------------------------------------------------------------------- |
+| `0`                    | IntValue within range                                                     |
+| `-9223372036854775808` | This is the lowest value that can be represented in the range             |
+| `9223372036854775807`  | This is the largest value that can be represented in the range            |
+| `"987654321"`          | A StringValue containing a base-10 representation of the value is allowed |
+
+These are invalid examples:
+
+| Output                  | Explanation                                                                                                                  |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `-10223372036854775808` | Value is lower than `-2^63`                                                                                                  |
+| `12223372036854775807`  | Value is greater than `2^63-1`                                                                                               |
+| `123.0`                 | FloatValue is not allowed                                                                                                    |
+| `"FFFFF"`               | A StringValue containing a base-16 representation of a valid value is not allowed                                            |
+| `"6543.000"`            | A StringValue containing a base-10 representation with a fractional part is not allowed. Even if the fractional part is zero |
+
+# Raw Input JSON spec
+
+For raw input the same rules apply as for Literal inputs.
+
+# References
+
+- [Support of `Long` (64-bit integer) scalar](https://github.com/graphql/graphql-spec/issues/73)
```

Comment:
I would say this is not spec material. The spec should be self sufficient. But my [logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/) is low

```suggestion
```